python/ray/scripts/scripts.py

```diff
@@ -1764,6 +1765,25 @@ def install_nightly(verbose, dryrun):
         subprocess.check_call(cmd)
 
 
+@cli.command()
+@click.option(
+    "--output", "-o", required=False, type=str, help="Output directory.")
+def get_cpp_example(output):
```

Comment:
Let's also add an option to only print the directory of headers and so files.

Reply:
Done.